Makefile

```
@@ -63,21 +62,13 @@ proxy-h:
 	consul connect proxy -service hivemetastore-local -upstream hive-metastore:9083 -log-level debug
 # to-minio
 proxy-m:
-	consul connect proxy -service minio-local -upstream minio:9000 -log-level debug
+	consul connect proxy -service minio-local -upstream minio:9090 -log-level debug
 # to-postgres
 proxy-pg:
 	consul connect proxy -service postgres-local -upstream postgres:5432 -log-level debug
-
 # to-presto
 proxy-presto:
 	consul connect proxy -service presto-local -upstream presto:8080 -log-level debug
 
-download-presto-cli:
-	curl https://repo1.maven.org/maven2/io/prestosql/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar -o ./${presto}
-	chmod +x ./${presto}
-
 presto-cli:
-ifeq (,$(wildcard ./${presto}))
-	$(MAKE) download-presto-cli
-endif
-	${presto} --server localhost:8080 --catalog hive --schema default --user presto
+	docker run --rm -it --network host prestosql/presto:${PRESTO_VERSION} presto --server localhost:8080 --http-proxy localhost:8080 --catalog hive --schema default --user presto --debug
```

Should this be 9090?
```suggestion
	docker run --rm -it --network host prestosql/presto:${PRESTO_VERSION} presto --server localhost:9090 --http-proxy localhost:9090 --catalog hive --schema default --user presto --debug
```

---

Why? Presto is default 8080. We could make use of a different port, but if so we should have a system.

like 10001 and onward.

---

Isn't this connecting a presto-cli to our presto that's running? The proxy is available at `localhost:9090`. Am I misunderstanding something